docs/testbed/README.testbed.VsSetup.md

```diff
@@ -38,36 +38,82 @@ sudo -H ./setup-management-network.sh
-2. Copy below image files to `~/veos-vm/images` on your testbed host:
+2. Copy below image files to location determined by `root_path` on your testbed host:
    - `Aboot-veos-serial-8.0.0.iso`
    - `vEOS-lab-4.20.15M.vmdk`
+
 ### Option 2: cEOS (container-based) image (recommended)
-#### Option 2.1: Download and import cEOS image manually
-Download the [cEOS image from Arista](https://www.arista.com/en/support/software-download). For example download file `cEOS64-lab-4.29.3M.tar`.
-Put the downloaded file under `~/veos-vm/images/` on your testbed host.
-The playbook will automatically look for the cEOS image file in the `~/veos-vm/images/` directory and import it into the testbed server.
 
-**Note**: *For time being, the image might be updated, in that case you can't download the same version of image as in the instruction,
-please download the corresponding version(following [Arista recommended release](https://www.arista.com/en/support/software-download#datatab300)) of image and import it to your local docker repository.
-The actual image version that is needed in the installation process is defined in the file [ansible/group_vars/all/ceos.yml](../../ansible/group_vars/all/ceos.yml), make sure you modify locally to keep it up with the image version you imported.*
+1. **Prepare folder for image files on test server**
 
-**Note**: *Please also notice the type of the bit for the image, in the example above, it is a standard 32-bit image. Please import the right image as your needs.*
-#### Option 2.2: Pull cEOS image automatically
-Alternatively, you can host the cEOS image on a http server. Specify `ceos_image_url` for downloading the image in file `ansible/group_vars/all/ceos.yml`. For example:
-```
-ceos_image_url: "http://example1.com/cEOS64-lab-4.29.3M.tar"
-```
-The `ceos_image_url` variable also can be a list of URLs, for example:
-```
-ceos_image_url:
-  - "http://example1.com/cEOS64-lab-4.29.3M.tar"
-  - "http://example2.com/cEOS64-lab-4.29.3M.tar"
-```
-The playbook will try to download the image from the URLs in the list one by one until it succeeds.
+    Create subfolder `images` under `root_path` defined in the `ansible/group_vars/vm_host/main.yml` file. Assume `root_path` is `veos-vm`:
+
+    ```bash
+    mkdir -p ~/veos-vm/images
+    ```
+
+2. **Prepare the cEOS image file**
+
+   #### Option 1.1: Manually download cEOS image
+
+   1. Obtain the cEOS image from [Arista's software download page](https://www.arista.com/en/support/software-download).
+   2. Place the image file under subfolder `images` of the location determined by `root_path` variable in `ansible/group_vars/vm_host/main.yml`:
+
+      Assume `root_path` is `veos-vm`:
+
+      ```bash
+      cp cEOS64-lab-4.29.3M.tar ~/veos-vm/images/
+      ```
+      The Ansible playbook for deploying testbed topology will automatically use the manually prepared image file from this location.
+
+   #### Option 1.2: Host the cEOS image file on a HTTP server
```

Comment:
This section is under **2**. I suppose it should be 2.2?

Reply:
Updated